code/stats/scoring.cpp

```diff
@@ -57,6 +59,17 @@ float Scoring_scale_factors[NUM_SKILL_LEVELS] = {
 	1.25f					// insane
 };
 
+traitor_override* get_traitor_override_pointer(const SCP_string& name)
+{
+	for (int i = 0; i < (int)Traitor_overrides.size(); i++) {
+		if (name == Traitor_overrides[i].name) {
```

Comment:
This should ideally be case-insensitive.  Most other string lookups, including ones set by sexps, are case-insensitive.

Reply:
If you are performing a *case-sensitive* check on a SCP_string, using == is perfectly reasonable.  But == won't work for case-insensitive checks.

And, separately, when performing case-insensitive checks, `SCP_string_lcase_equal_to()()` is favored over `stricmp` because it avoids unnecessarily converting both SCP_strings to C-strings.